docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 version: "3"
 services:
   hapi-fhir-jpaserver-start:
+    network_mode: host
     build: .
     container_name: hapi-fhir-jpaserver-start
     restart: on-failure
     ports:
       - "8080:8080"
-  hapi-fhir-mysql:
```

Comment:
I don't think we should be removing the MySQL datasource from here. Ideally, the `docker-compose` should start all services needed for local development and testing.

Reply:
ok I will revert that changes